demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -2116,6 +2118,19 @@ def find_dependencies_manager(
             dependent_packs = json.dumps(dependent_packs, indent=4)
             click.echo(click.style(dependent_packs, bold=True))
 
+        elif dependency:
+            dependent_packs, _ = get_packs_dependent_on_given_packs(input_paths, id_set_path,  # type: ignore[arg-type]
+                                                                    output_path, verbose=True)
+            dependent_items = dependent_packs[Path(input_paths[0]).name].get('packsDependentOnThisPackMandatorily')  # type: ignore[attr-defined]
+            if dependency in dependent_items:
+                print_success(f"The pack \"{dependency}\" depends on \"{Path(input_paths[0]).name}\" "
+                              f"with the following items:")
```

Comment:
`input_pack_name` is depndent in `dependency` and not the other way around 
```suggestion
                print_success(f'The pack "{input_pack_name}" depends on "{dependency}" '
                              f'with the following items:')
```